app/src/main/java/woowacourse/movie/activity/detail/contract/presenter/MovieDetailPresenter.kt

```diff
@@ -0,0 +1,83 @@
+package woowacourse.movie.activity.detail.contract.presenter
+
+import domain.movieinfo.MovieDate
+import domain.movieinfo.MovieTime
+import domain.screeningschedule.ReservationDate
+import woowacourse.movie.activity.detail.contract.MovieDetailContract
+import woowacourse.movie.dto.movie.MovieUIModel
+import woowacourse.movie.dto.movie.TheaterUIModel
+import woowacourse.movie.dto.ticket.TicketCountUIModel
+import woowacourse.movie.mapper.movie.mapToUIModel
+import woowacourse.movie.mapper.ticket.mapToDomain
+import woowacourse.movie.mapper.ticket.mapToUIModel
+import java.time.LocalDate
+
+class MovieDetailPresenter(
+    private val view: MovieDetailContract.View,
+    private var dateSpinnerPosition: Int = 0,
+    private var timeSpinnerPosition: Int = 0,
+) :
+    MovieDetailContract.Presenter {
```

Comment:
```suggestion
) : MovieDetailContract.Presenter {
```
콜론과 같은 라인에 정의하는 것이 컨벤션입니다.